spec/requests/v1/mixtape_request_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'mixtape endpoints' do
+  describe 'POST #create' do
+    it 'returns JSON for a single mixtape' do
+      artist = create(:artist)
+      songs = create_list(:song, 5, artist: artist)
+
+      post(mixtapes_url, {no_song: 4,artist: artist}.to_json , accept_headers)
```

Comment:
I have it running from the command line. And I didn't run it before this PR because I was in a rush, but it would be good to set it up in Sublime. 

I also realize that this PR is not comparing with my master branch but rather with Helen's branch. 

@dthyresson